package.json

```diff
@@ -35,7 +35,8 @@
     "test:journey:run-wasm": "jest --detectOpenHandles journey/entrypoint-wasm.test.ts",
     "test:journey:upgrade": "npm run test:journey:k3d && npm run test:journey:image && jest --detectOpenHandles journey/pepr-upgrade.test.ts",
     "format:check": "eslint src && prettier src --check",
-    "format:fix": "eslint src --fix && prettier src --write"
+    "format:fix": "eslint src --fix && prettier src --write",
```

Comment:
Based off the [original issue](https://github.com/defenseunicorns/pepr/issues/1105), we should find a way to locally lint _just_ the staged files. To see how long the linting takes locally, use the command `npm run format:fix | awk '{sum += $2} END {print sum}'`.

For example, on my machine linting all the files takes `418ms`. That's not bad, but we can be more efficient if we configure a tool like [lint-staged](https://github.com/lint-staged/lint-staged).

Reply:
Sure Thing!!!